src/endpoint/sts/ops/sts_post_assume_role.js

```diff
@@ -35,7 +35,7 @@ async function assume_role(req) {
                 Credentials: {
                     AccessKeyId: access_keys.access_key.unwrap(),
                     SecretAccessKey: access_keys.secret_key.unwrap(),
-                    Expiration: expiry,
+                    Expiration: Date.now() + expiry,
```

Comment:
I understand the fix - to have a timestamp as in the API  [AssumeRole](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html) (in the type [Credentials](https://docs.aws.amazon.com/STS/latest/APIReference/API_Credentials.html)).

I'm concerned about something else - if you create the expiration timestamp in the json reply part - where do you store it? how do you verify that after an hour this role does not have access?